app/controllers/users_controller.rb

```diff
@@ -0,0 +1,10 @@
+class UsersController < ApplicationController
+  def show
+    @user = User.find(params[:id])
+  end
+
+  def posts
+    @user = User.find(params[:id])
```

Comment:
- [ ] [REQUIRED] Good job so far! The `users_controller.rb` should contains with `#index` and `#show` actions, please change the `posts` 🙏🏼 

```ruby
class PostsController < ApplicationController
  
  def index; end

  def show; end

end
```

Reply:
Fixed.